arangod/Statistics/StatisticsWorker.cpp

```diff
@@ -924,6 +924,12 @@ void StatisticsWorker::generateRawStatistics(VPackBuilder& builder, double const
   builder.add("server", VPackValue(VPackValueType::Object));
   builder.add("uptime", VPackValue(serverInfo._uptime));
   builder.add("physicalMemory", VPackValue(TRI_PhysicalMemory));
+  builder.add("transactions", VPackSlice::emptyObjectSlice());
```

Comment:
This will *not* work.
```suggestion
  builder.add("transactions", VPackValue(VPackValueType::Object));
```